tests/devapps/DesktopTestApp/MainForm.cs

```diff
@@ -116,12 +116,6 @@ private void settings_Click(object sender, EventArgs e)
             tabControl1.SelectedTab = settingsTabPage;
         }
 
-        private void cache_Click(object sender, EventArgs e)
```

Comment:
Are we removing the cache page from here?
it is pretty useful for testing

Reply:
Added back.